lib/dfe/analytics/entity_table_check_job.rb

```diff
@@ -26,17 +26,54 @@ def perform
       end
 
       def entity_table_check_data(model)
+        adapter_name = ActiveRecord::Base.connection.adapter_name.downcase
+
+        if adapter_name != 'postgresql' && Rails.env.production?
+          Rails.logger.info('DfE::Analytics: Entity checksum: Only Postgres databases supported on PRODUCTION')
+          return
+        end
+
```

Comment:
Might be safer to get the current timestamp like this:

``` Ruby
checksum_calculated_at = 
  ActiveRecord::Base.connection.execute('SELECT CURRENT_TIMESTAMP(6)')[0]['current_timestamp']
```

Reply:
@asatwal I had to adapt this a little to make it compatible with SQLite3 as well as Postgres, kept the concept though